packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -535,6 +535,23 @@ export default class ScriptTransformer {
   }
 }
 
+export const createTranspilingRequire = (config: Config.ProjectConfig) => {
```

Comment:
```suggestion
export function createTranspilingRequire(config: Config.ProjectConfig) {
```

Reply:
Any particular reason? Surrounding functions use arrows, which is why I used that.

Reply:
less syntax, and we don't need the lexical `this`